spec/models/section_spec.rb

```diff
@@ -142,6 +142,24 @@ def key_classes_for(hash)
     end
   end
 
+  describe '#save' do
+    it 'saves the last two editions' do
```

Comment:
I notice that the spec name has changed from "saves the the two most recent editions" to "saves the last two editions". Was this intentional? The new spec name seems more accurate in a way, although it does make me wonder whether these editions should be explicitly ordered the same way as when the section is built. At the very least I think this change merits a mention in the commit note.